app/controllers/tax_returns_controller.rb

```diff
@@ -0,0 +1,54 @@
+class TaxReturnsController < ApplicationController
+  # Because these aren't CRUD verb methods, they need to be explicitly listed on the before actions
+  before_action :block_access_on_production, only: [:authorize_signature, :sign, :success]
+  before_action :load_tax_return, except: [:success]
+  before_action :check_for_forms, only: [:sign, :authorize_signature]
+
+  def authorize_signature
+    tax_return = TaxReturn.find(params[:tax_return_id])
```

Comment:
would it be a good idea to do something if we can't find the tax return? genuinely not sure 🤔